### PR TITLE
ci: github action to run 'make test' on every pr

### PR DESCRIPTION
**Pull request by leelavg**
Signed-off-by: Leela Venkaiah G <lgangava@redhat.com>

### CODE REVIEW

File: .github/workflows/on-pr-submit.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+on:
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - 'doc/**'
+
+jobs:
+  envtest:
+    name: envtest
```
Comment (nbalacha) [inline .github/workflows/on-pr-submit.yaml:13]: please rename this to  'test" both here and in the job name

**Comment by leelavg** [comment]
> Please replace envtest there as well.

- I did see that actually and at the same time, current action has only envtest so didn't change that for easy ref
- Still want me to change?

**Comment by nbalacha** [comment]
> > Please replace envtest there as well.
> 
> * I did see that actually and at the same time, current action has only envtest so didn't change that for easy ref
> * Still want me to change?

Yes.

**Review by jmolmo — APPROVED** [review]

**Review by nbalacha — APPROVED** [review]
/lgtm

**Pull request merged**